lms/services/blackboard_api/service.py

```diff
@@ -45,6 +46,23 @@ def get_token(self, authorization_code):
             response.validated_data.get("expires_in"),
         )
 
+    def list_files(self, course_id):
+        """
+        Return the list of files in the given course.
+
+        :raise ProxyAPIAccessTokenError: if we don't have a Blackboard API
+            access token for the current user
+        """
+        return self._http_service.get(
+            self._api_url(f"courses/uuid:{course_id}/resources"),
+            oauth=True,
+            schema=BlackboardListFilesSchema,
+        ).validated_data
```

Comment:
Would it make sense that `list_files` returned `List[FileSchema]` I'd have thought that would be straightforward thing to do but can figure it out on the docs.